### DISCUSSION TITLE
Issue in auto mapper for version 2.11.1 with abp io

### DISCUSSION THREAD

**Discussion by alaawahbah** [elsa-workflows/elsa-core#3983] [Show and tell]
Why the papping doesn't exist in the packages?

Solution

<PackageReference Include="Elsa" Version="2.11.1" />
		<PackageReference Include="Elsa.Activities.Email" Version="2.11.1" />
		<PackageReference Include="Elsa.Activities.Http" Version="2.11.1" />
		<PackageReference Include="Elsa.Server.Api" Version="2.11.1" />

add above packages in the application service project
then add 

CreateMap<WorkflowDefinition, WorkflowDefinitionSummaryModel>();
        CreateMap<WorkflowInstance, WorkflowInstanceSummaryModel>();
        CreateMap<WorkflowBlueprint, WorkflowBlueprintSummaryModel>();
        CreateMap<WorkflowBlueprint, WorkflowBlueprintModel>();
        CreateMap<ActivityBlueprint, ActivityBlueprintModel>();
        CreateMap<WorkflowDefinition, WorkflowDefinitionVersionModel>();
        CreateMap<Elsa.Services.Models.Connection, Elsa.Server.Api.Endpoints.WorkflowRegistry.ConnectionModel>();

in your mapping profile

**Comment by sfmskywalker** [comment]
Thanks for sharing this tidbit of information 👍🏻
